getting-started/devflowapp/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.13
+FROM golang:1.14
```

Comment:
This seems to work but I don't know what considerations there are on this app updating.